### ISSUE TITLE
`skip_empty_diffs` config does not work

### ISSUE THREAD

**Issue by ilyagr** [sachaos/viddy#119]
I set up my `viddy.toml` as follows:

```toml
[general]
shell = "echo"  # To check that this is the correct config file path
skip_empty_diffs=true
```

However, the skip_empty_diffs option did not take. Here's the result of `viddy abracadabra` with that config:

![image](https://github.com/sachaos/viddy/assets/4123047/7bbbb5e2-6dfd-439d-94d1-bb1471a5c6b1)

I'm using viddy 0.4.0 installed form Nix, on Mac OS Sonoma 14.5, Apple Silicon

I'm not sure what's going on, my understanding of https://github.com/sachaos/viddy/commit/106e52d8f0ff8ce8e25dd97c2f65670289cec81a#diff-0e426a43248661127a0c0ee115aef7a1093b635f8993b3f7ebb1dd9f05b8f249R153 is that the option should work, which is why I filed #118.

**Comment by sachaos** [comment]
This bug fixed in viddy v1.0.0-rc.1 
https://github.com/sachaos/viddy/issues/117#issuecomment-2291069041

**Comment by sachaos** [comment]
Viddy v1.0.0 is available now.
